example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -12,6 +12,7 @@
 		3B80C3941E831B6300D905FE /* App.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3B80C3931E831B6300D905FE /* App.framework */; };
 		3B80C3951E831B6300D905FE /* App.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 3B80C3931E831B6300D905FE /* App.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		51D6FC297CD69581F379F1A5 /* Pods_Runner.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = E2CDB95AAF3CDEFF4B244175 /* Pods_Runner.framework */; };
+		6E6CC354239B7697003D583C /* AirshipConfig.plist in Resources */ = {isa = PBXBuildFile; fileRef = 6E6CC353239B7697003D583C /* AirshipConfig.plist */; };
```

Comment:
Doesn't need this. This ref will cause a build failure unless we update the job to inject the plist.

Reply:
Just delete the greyed-out reference in the project explorer and that should fix it.